client/CVideoHandler.cpp

```diff
@@ -10,6 +10,7 @@
 #include "StdInc.h"
 #include "CVideoHandler.h"
 
+#include "CMT.h"

```

Comment:
is this needed here?

Reply:
Yes.  Change due to change in SDL_Extensions.h, which is included in VideoHandler

Previously globals such as screen were defined in both CMT.h and SDL_Extensions.
Since I wanted to detect usage of these globals I removed them from SDL_Extensions, now they are defined only in CMT.